lib/observe.js

```diff
@@ -82,6 +82,9 @@ Observer.prototype._onOperation = function (doc) {
 };
 
 Observer.prototype._checkKuery = function (coll) {
+  if (!this._kuery) {
+    this._kuery = new Kuery(this._query.query);
+  }
```

Comment:
Where is `this._checkKuery` called from? Might be a scope issue, with `this` not referring to the expected thing. In that case, we would get follow-up problems with accessing `this._query.query` as well.

Reply:
https://github.com/surikaterna/viewdb_persistence_store_mongodb/blob/4de1e51bb6f727966a6580cfbe81e0f776a14d12/lib/observe.js#L26-L28
I should be the same scope as it always called from the listener